py-rse/bash-basics.Rmd

```diff
@@ -870,15 +870,10 @@ $ ls data
 ```
 
 ```text
-anne_of_green_gables.txt        moby_dick.txt
-common_sense.txt                mysterious_affair_at_styles.txt
-count_of_monte_cristo.txt       pride_and_prejudice.txt
-dracula.txt                     README.md
-emma.txt                        sense_and_sensibility.txt
-ethan_frome.txt                 sherlock_holmes.txt
-frankenstein.txt                time_machine.txt
-jane_eyre.txt                   treasure_island.txt
-life_of_frederick_douglass.txt
+README.md         moby_dick.txt
```

Comment:
Whether upper case names appear first depends on what `LS_COLLATE` is set to. It at least used to be the default on Ubuntu that upper case is before lower case, but I haven't installed that distro in a while so I am not sure about the current state of things. I tested on a friend's mac and there upper case went first. Github sorts upper case first as well.